WordPress/Classes/Services/PostCoordinator.swift

```diff
@@ -408,21 +465,9 @@ class PostCoordinator: NSObject {
     }
 
     /// Try to cancel the current sync operation, which is not always possible.
-    private func tryCancelSyncOperation(_ operation: SyncOperation, latest: AbstractPost) {
-        // If there is a current sync operation running and it doesn't target
-        // the latest revision, then try to cancel it.
+    private func tryCancelSyncOperation(_ operation: SyncOperation) {
         switch operation.state {
         case .uploadingMedia:
-            // Cancel the upload for media, but keep the tasks scheduled
-            // for the media still present in the revision.
-            let deleted = Set(operation.revision.media).subtracting(Set(latest.media))
```

Comment:
I remove it for now because the coordinator is not well suited to handle multiple revisions – keeping it simple.